src/test_l1_dsHdmiIn.c

```diff
@@ -2254,10 +2316,10 @@ void test_l1_dsHdmiIn_negative_dsGetEdid2AllmSupport(void) {
     UT_ASSERT_EQUAL(dsHdmiInInit(), dsERR_NONE);
```

Comment:
Why CHECK_FOR_EXTENDED_ERROR_CODE is not for the dsERR_NOT_INITIALIZED enhanced case?

Reply:
There is no API definition in HAL, hence not handled as the old error code was unknown

Reply:
done